packages/sample-app/sample_app/strands_example.py

```
@@ -0,0 +1,410 @@
+    """Get weather information for travel planning."""
+    weather_info = f"Weather forecast for {city}: Generally pleasant with temperatures 20-25°C. {dates} looks good for travel with sunny skies and light winds."
+    add_to_memory("research_agent", "weather_check", {"city": city, "dates": dates, "forecast": weather_info})
+    return weather_info
+
+
+# Trip Planning Agent Tools
+@tool
+def create_itinerary(destination: str, duration: str, interests: str) -> str:
+    """Create detailed travel itinerary based on research and preferences."""
+    # Check memory for research data
+    research_data = shared_memory.get("research_data", {})
```

Ruff (0.12.2)

Local variable `research_data` is assigned to but never used

Remove assignment to unused variable `research_data`

(F841)